heatmapchart/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "@perses-dev/heatmap-chart",
+  "version": "0.7.0",
+  "scripts": {
+    "dev": "rsbuild dev",
+    "build": "rsbuild build",
+    "lint": "eslint src --ext .ts,.tsx",
+    "test": "cross-env LC_ALL=C TZ=UTC jest",
+    "type-check": "tsc --noEmit"
```

This will help building javascript module variants
```suggestion
    "dev": "rsbuild dev",
    "build": "npm run build-mf && concurrently \"npm:build:*\"",
    "build-mf": "rsbuild build",
    "build:cjs": "swc ./src -d dist/lib/cjs --strip-leading-paths --config-file ../.cjs.swcrc",
    "build:esm": "swc ./src -d dist/lib --strip-leading-paths --config-file ../.swcrc",
    "build:types": "tsc --project tsconfig.build.json",
    "lint": "eslint src --ext .ts,.tsx",
    "test": "cross-env LC_ALL=C TZ=UTC jest",
    "type-check": "tsc --noEmit"
```